server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const path = require('path')
+const app = express()
+const port = process.env.PORT || 5001
+
+// Dynamic configuration for openshift API calls
+app.get('/config.js', (req, res) => {
+  // The x-forwarded-access-token header will be forwarded
+  // by the oauth-proxy to this server.
+  // The below makes this token available to the client app
+  // for use in API calls to the configured OpenShift cluster.
+  res.send(`window.OPENSHIFT_CONFIG = {
+    openshiftAPIUrl: "${process.env.OPENSHIFT_API_URL}",
+    accessToken: "${req.headers['x-forwarded-access-token']}"
```

Comment:
Am I correct in saying both of these values have the possibility of being undefined? If they are this will return a string `"undefined"` instead of something like `null`. We should probably return `null`. wdyt?